study.md

```diff
@@ -151,7 +157,10 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+1. GET instructs the server to transmit the data identified by the URL to the client.
+2. A PUT request is used when you wish to create or update the resource identified by the URL
+3. DELETE should be used when you want to delete the resource identified by the URL of the request
+4. POST is used when the processing you wish to happen on the server should be repeated.
```

Comment:
POST is to create a new resource while PUT and PATCH are to update or modify an existing resource